src/adapters/swagger20.es6

```diff
@@ -157,8 +164,11 @@ export function parse({ source }, done) {
         schemaAsset.attributes.contentType = 'application/schema+json';
       }
 
+      // Currently, default responses are not supported in API Description format
+      let relevantResponses = _.omit(methodValue.responses, 'default');
```

Comment:
It is the default value for a response. See https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#fixed-fields-9